### DISCUSSION TITLE
PyScript is hard to "find out"

### DISCUSSION THREAD

**Discussion by fpliger** [pyscript/pyscript#1886] [General]
Opening this as a follow up discussion to our community call topic: "It's hard to find out resources around PyScript"

Issue here is that it's hard for users to find where to find resources, examples, etc... also hard to find when the community 

* [ ] Community calls are not well advertised and hard to find
* [ ] Discord server is not well advertised and visible 
* [ ] Calls are not being uploaded to the Youtube Channel
* [ ] YouTube channel is not well advertised and linked somewhere easy to find

@tedpatrick @ntoll @WebReflection @JeffersGlass  Please add if I forgot some things...

**Comment by tedpatrick** [comment]
Several aspects of this are:
- centralized url (pyscript.net) where these "events" are listed.

- informing those interested in PyScript with an email. Rather than me visiting pyscript.net/twitter/docs/discourse, send an email newsletter. Keep me informed. Odds are high that 95% of those interested in PyScript have yet to learn about the latest release.

- the team is small and busy. This all needs to be automated; we have failed if it depends on Fabio/Nicholas doing something manually.

The context is interesting. When I was inside Anaconda on PyScript, I always knew when and where things were happening. Now that I am outside... crickets.

**Comment by ntoll** [comment]
**EDIT** - grumpiness ahead. Patience advised.

OK... so **this is a classic example of how we're doing it wrong** - and I feel deeply frustrated by it (clearly I've woken up on the wrong side of the bed, sorry). :facepalm: 

Respectfully, there is **already** a discussion about this, within the context of community engagement over on discord.

https://discord.com/channels/972017612454232116/1180095453111533638

(BTW, this was also mentioned in the community call.)

I feel we're going round and round in circles because now we have a discussion in two places. I thought discussions about the technical aspects of PyScript should happen here on GitHub (i.e. close to the code), and community discussions happen on discord (i.e. close to where the community hangs out)..?

It's great we want to engage in these aspects of the project, but I want to throw up my hands in frustration, and say, "I give up" because we can't even engage in a coordinated manner about coordinating engagement. It's like watching an episode of the Keystone Cops (ref: https://www.youtube.com/watch?v=GwUUUqgfu-E - and I'm just as guilt as anyone else of this). :wink: 

So, **I'll be discussing this over in discord** because that's where I believe we should be doing this. 

I'll happily discuss technical things here on GitHub. 

But I passionately believe this is not the place for community engagement related discussions because it is hidden from the very community with whom we want to engage.

**Reply by WebReflection** [reply]
I agree with you but if anything, here is where people land and I think there's something actionable already:

  * the [Resources have broken links](https://github.com/pyscript/pyscript#resources) ... the *Official Docs* in particular ... this is on us
  * I would personally move the resources on the top of the Readme, not at the end ... that makes it easier to reach us
  * I don't understand why we still have stuff like "*This is an extremely experimental project, so expect things to break!*" or "*PyScript has been only tested on Chrome at the moment.*" which are both wrong / bad statements to me ... **remove the notes** as we work in Safari, Firefox and Chrome and Edge and every other Chromium based browser + mobile
  * there's room for improvements even in the calendar and community calls or events ... I clicked the link and couldn't understand much in there

Beside that, the Zoom link (I've double checked) is indeed always the same, at least these days, but it's nowhere in the README.

The Discord feedback is great to read and engage with, but there are obvious low-hanging fruits to me here too.

**Comment by tedpatrick** [comment]
As a medium:
 - Discord is a river. Messages float by and are lost in time, even threads
get lost in time.
 - GitHub discussions are a lake. Discussion threads are permanent and
serve as a great archival medium, but it is developer-centric.

I feel we just need to pick the right medium for the job.

1. Long-term developer-centric discussion with potential for GH Issue
creation? GH Discussions
2. Short-term discussion with the community? Discord

I digress.

**Comment by ntoll** [comment]
Just wanted to apologise for my grumpy reply. Believe me, I'm not having a good day today... and while this may explain, it is no excuse for such grumpiness.

**Comment by fpliger** [comment]
@ntoll  sorry about your bad day and frustration...  big hugs your way! 😉 

I think there is some confusion here. Fwiw there as a difference between  discussions related to actionable items (like, in this case, we had already discussed many of these things during the community call and my intention was mainly to keep the todo list here) vs discussions in support of community or discussing WITH the broader community (== Discord). Honestly, I find Discord extremely distracting, confusing, and really hard to use as a place to store discussions for posterity. (It may be only me though! Happy to be educated about it as a tool but...) As it is, I'd definitely be against us keeping it as the place we discuss important decisions, for the reasons @tedpatrick  mentioned above.

Totally fine with me to keep some of the discussion related to the tasks above on Discord and keep this discussion as the place where we actually keep the tasks and can track their completion .
